configure/arbiter.md

```diff
@@ -0,0 +1,9 @@
+Enabling Replica Set Arbiter nodes 
+==============================================================
+
+Percona Server for MongoDB [replication model](https://www.percona.com/blog/2018/05/17/mongodb-replica-set-transport-encryption-part-1/) is based on elections, during which nodes of the Replica Set [choose the one of them](https://docs.mongodb.com/manual/core/replica-set-elections/#replica-set-elections) to become a primary node. Elections are the reason to avoid the even number of nodes in the cluster, and that's why the cluster should have at least 3 nodes. Normally each node stores a complete copy of the data, but there is also a possibility to reduce disk space used by the database still remaining the odd nodes number for elections. [Replica Set Arbiter nodes](https://docs.mongodb.com/manual/core/replica-set-arbiter/) are special members of the cluster which do not replicate data, but only vote in elections for primary. Obviously such empty node doesn't demand huge disk space.
```

Comment:
```suggestion
Percona Server for MongoDB [replication model](https://www.percona.com/blog/2018/05/17/mongodb-replica-set-transport-encryption-part-1/) is based on elections, during which nodes of the Replica Set [choose the one of them](https://docs.mongodb.com/manual/core/replica-set-elections/#replica-set-elections) to become a primary node. Elections are the reason to avoid the even number of nodes in the cluster, and that's why the cluster should have at least 3 nodes. Normally each node stores a complete copy of the data, but there is also a possibility to reduce disk IO and space used by the database still remaining the odd member number for elections. [Replica Set Arbiter nodes](https://docs.mongodb.com/manual/core/replica-set-arbiter/) are special members of the cluster which do not replicate data, but only vote in elections for the primary. Obviously, such empty node doesn't demand any persistent volume.
```